tests/conftest.py

```diff
@@ -89,7 +89,7 @@ def memory_jobstore():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def log_to_stdout():
+def log_to_stdout_auto_use():
```

Comment:
This function was name-clashing with another fixture of the same name in this file. Since this one has `autouse=True`, I renamed this one since the name shouldn't matter?

Reply:
Yeah, that looks good.